sys/kern/uipc_usrreq.c

```diff
@@ -631,7 +631,7 @@ uipc_bindat(int fd, struct socket *so, struct sockaddr *nam, struct thread *td)
 	UNP_PCB_UNLOCK(unp);
 
 	buf = malloc(namelen + 1, M_TEMP, M_WAITOK);
-	bcopy(soun->sun_path, buf, namelen);
+	bcopy(__bounded_addressof(soun->sun_path, namelen), buf, namelen);
```

Comment:
This seems unnecessary?

Reply:
Yes, can't remember why I added it, might be a leftover from `__no_subobject_bounds` annotation in sockaddr_un that ultimately got removed.